python-backend/call_wrapper.py

```diff
@@ -0,0 +1,56 @@
+import logging
```

Comment:
introduced this wrapper so we can set a timeout on calls to ResearchTools and LIReC